cohttp-eio/src/rwer.ml

```diff
@@ -0,0 +1,65 @@
+(* This modules encapsulates refactored - common - readers and writers
+   used by the Client and Server modules.
+
+    rwer.ml => (R)eader (W)riter + er
+*)
+
+module Buf_read = Eio.Buf_read
+module Buf_write = Eio.Buf_write
+
+let return v _ = v
```

Comment:
Note: we have `Eio.Buf_read.return` now.

Reply:
Done. Thanks.